### PR TITLE
[W5.4][T13-A1]Sirisha Kommareddi

### PR DESCRIPTION
**Pull request by siri99**
pull request

### CODE REVIEW

File: src/seedu/addressbook/commands/FindCommand.java [hunk 1]
```diff
@@ -49,12 +50,41 @@ public CommandResult execute() {
...
         return matchedPersons;
     }
 
+
+
+    // Method to convert all words to lowercase so that the comparison can be case-insensitive
+    public Set<String> changeSetElements(Set<String> s)
+    {
+        Set<String> temp= new HashSet<String>();
+        for(String i : s)
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/FindCommand.java]: coding standard violation. please check how to write loops with single line of code inside

File: src/seedu/addressbook/commands/FindCommand.java [hunk 2]
```diff
@@ -49,12 +50,41 @@ public CommandResult execute() {
...
             }
         }
         return matchedPersons;
     }
 
+
+
+    // Method to convert all words to lowercase so that the comparison can be case-insensitive
+    public Set<String> changeSetElements(Set<String> s)
+    {
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/FindCommand.java]: the bracket should follow the function signature not next line

File: test/java/seedu/addressbook/commands/FindCommandTest.java [hunk 3]
```diff
@@ -25,8 +25,11 @@ public void execute() throws IllegalValueException {
         //same word, same case: matched
         assertFindCommandBehavior(new String[]{"Amy"}, Arrays.asList(td.amy));
 
-        //same word, different case: not matched
-        assertFindCommandBehavior(new String[]{"aMy"}, Collections.emptyList());
+        //same word, different case: not matched - CASE INSENSITIVE
+        assertFindCommandBehavior(new String[]{"aMy"}, Arrays.asList(td.amy));
+
+        //original line
+        //assertFindCommandBehavior(new String[]{"aMy"}, Collections.emptyList());
```
Comment (okkhoy) [inline test/java/seedu/addressbook/commands/FindCommandTest.java]: don't retain commented out code; delete it. you can always retrieve the lines back from VCS

File: doc/UserGuide.md [hunk 4]
```diff
@@ -57,80 +57,84 @@ Format: `list`
 Finds persons whose names contain any of the given keywords.<br>
 Format: `find KEYWORD [MORE_KEYWORDS]`
 
-> The search is case sensitive, the order of the keywords does not matter, only the name is searched, 
-and persons matching at least one keyword will be returned (i.e. `OR` search).
+> The search is NOT case-sensitive, the order of the keywords does not matter, only the name is searched,
```
Comment (okkhoy) [inline doc/UserGuide.md]: you may want to update the "one line description" above to reflect find by tag?

**Comment by okkhoy** [comment]
@siri99 
Some comments added. Please ack & close the PR after reading comments.

**Comment by siri99** [comment]
All comments noted. Will change code accordingly. Thanks

**Pull request closed without merging**